packages/util/src/constants.ts

```diff
@@ -64,6 +64,17 @@ export const KECCAK256_RLP_S = '0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cad
  */
 export const KECCAK256_RLP = hexToBytes(KECCAK256_RLP_S)
 
+/**
+ * Keccak-256 hash of the RLP of an empty requests hash
+ */
+export const KECCAK256_RLP_RH_S =
+  '0x6036c41849da9c076ed79654d434017387a88fb833c2856b32e18218b3341c5f'
```

Comment:
Could you clarify this "magic constant" by explicitly calling the hash function with the input? I also think this should be SHA256? For the empty hash (or also all other hashes) this is not the RLP of the input (the input are opaque bytes which are concatenated).

```suggestion
  export const SHA256_EMPTY_RH_S = sha256(new Uint8Array([0, 1, 2]))
```

(You will need to also import sha256 likely)